_sources/functions_in_cpp/Exercises.rst

```diff
@@ -0,0 +1,314 @@
+Multiple Choice Exercises
+-------------------------
+
+Answer the following **Multiple Choice** questions to
+practice what you have learned in this chapter.
+
+.. mchoice:: functions_in_cpp_mcq_1
+    :practice: T
+
+    What should be the return type of the function ``convertToCelsius``?
+
+    .. code-block:: cpp
+
+      ______ convertToCelsius (double fahrenheit) {
+        double celsius;
+        celsius = (fahrenheit - 32) * 5 / 9;
+        return celsius;
+      }
+
+    - ``int``
+
+      - What variable are we returning in the function, and what is the variable's type? 
+
+    - ``double``
+
+      + The function returns ``celsius``, which is a ``double``.
+
+    - ``string``
+
+      - What variable are we returning in the function, and what is the variable's type? 
+
+    - ``void``
+
+      - Since we are returning something in the function, the function is not ``void``.
+
+.. mchoice:: functions_in_cpp_mcq_2
+    :practice: T
+
+    What would be returned by ``secretFunction`` if the input was 14?
+
+    .. code-block:: cpp
+
+      int secretFunction (int input) {
+        if (input % 2 == 0) {
+          return 3 * input - 2;
+        }
+        else {
+          if (input % 7 == 0) {
+            return input;
+          }
+          return 2 * input + 9;
+        }
+        return input + 4;
+      }  
+
+    - 14
+
+      - Although 14 is divisible by 7, take another look at the conditionals. 
+
+    - 18
+
+      - The flow of code would never reach the last return statement.
+
+    - 36
+
+      - Check your order of operations! 
+
+    - 37
+
+      - Take a closer look at the conditional statements. 
+
+    - 40
+
+      + Since 14 is divisible by 2, the function returns two less than three times 14.
+
+.. mchoice:: functions_in_cpp_mcq_3
+    :practice: T
+
+    If we wanted to create a boolean function called ``isPrime``, which takes an ``int input``
+    as a parameter, which of the following would be the correct function header?
```

Comment:
Let's use this wording instead: "If we wanted to create a function called ``isPrime`` that returns a ``bool`` and takes an ``int input``..."